cpp/include/cudf/lists/contains.hpp

```diff
@@ -37,8 +37,6 @@ namespace lists {
  * Output `column[i]` is set to null if one or more of the following are true:
  *   1. The search key `search_key` is null
  *   2. The list row `lists[i]` is null
- *   3. The list row `lists[i]` does not contain the search key, and contains at least
```

Comment:
I thought I had removed this line as part of #9510. Thanks for cleaning this up.